python/ray/_private/function_manager.py

```diff
@@ -124,15 +118,37 @@ def compute_collision_identifier(self, function_or_class):
         # Return a hash of the identifier in case it is too large.
         return hashlib.sha1(collision_identifier.encode("utf-8")).digest()
 
+    def check_local(self,module_name,function_class_name):
```

Comment:
I'm a bit worried about the performance of this function. Can we add a local cache here?

Reply:
Just checked, this should be fine. Loading already have a cache. Exporting will be only done once.